core/src/erasure.rs

```diff
@@ -271,7 +116,7 @@ impl CodingGenerator {
                 data_blobs
                     .iter()
                     .fold(0, |max, blob| cmp::max(blob.read().unwrap().meta.size, max)),
-                wb()
+                align()
```

Comment:
this is a misnomer, name should include something about length, since the length alignment and buffer alignment requirements are different